src/procedures/proc_fulltext_query.c

```diff
@@ -44,7 +44,8 @@ ProcedureResult Proc_FulltextQueryNodeInvoke(ProcedureCtx *ctx, const SIValue *a
 	Index *idx = Schema_GetIndex(s, NULL, IDX_FULLTEXT);
 	if(!idx) return PROCEDURE_ERR; // TODO this should cause an error to be emitted.
 
-	QueryNodeContext *pdata = rm_malloc(sizeof(QueryNodeContext));
+	ctx->privateData = rm_malloc(sizeof(QueryNodeContext));
+	QueryNodeContext *pdata = ctx->privateData;
```

Comment:
No cast needed?

Reply:
`void *`s like `ctx->privateData` can always be converted without an explicit cast.